java/src/main/java/com/powsybl/python/commons/PyPowsyblApiHeader.java

```diff
@@ -623,7 +623,14 @@ public enum ElementType {
         INJECTION,
         BRANCH,
         TERMINAL,
-        SUB_NETWORK;
+        SUB_NETWORK,
+        AREA,
+        AREA_VOLTAGE_LEVELS,
+        AREA_BOUNDARIES,
+        AREA_ADD_VOLTAGE_LEVELS,
+        AREA_REMOVE_VOLTAGE_LEVELS,
+        AREA_ADD_BOUNDARIES,
+        AREA_REMOVE_BOUNDARIES;
```

Comment:
this is quite tweaking the `ElementType` concept ... please let me know if this is OK of if must be done differently...

Reply:
Nok for me, this is really not the purpose of the dataframes. Dataframes contains some data, not operations, and this is the way we use data (though a creation, modif or read method) that we express the operation.

Reply:
yes it was a failed attempt at solving https://github.com/powsybl/pypowsybl/pull/840#issuecomment-2373259730, but thinking about it, we don't redefine areas that often, so I reverted to something simpler